Sources/Swift/Protocol/SentryLogLevel.swift

```diff
@@ -0,0 +1,28 @@
+extension SentryLog {
+    enum Level: String, Codable {
```

Comment:
`m`: What drove you to add this as an extension? I think defining it directly in SentryLog struct would be better for readability. The same applies to the `enum Attribute`.